.gitattributes

```diff
@@ -0,0 +1 @@
+sdk/**/* linguist-generated=true
```

Comment:
I included this to help keep PRs a little less noisy. It has the effect of automatically collapsing generated SDK files in GitHub code reviews. They're still available for review, they just aren't automatically mixed in with real code changes.

I find it much nicer to deal with PRs when generated code is collapsed, but I'm happy to omit this if we prefer the default behavior of expanding everything.